packages/suite/src/components/wallet/TransactionItem/TransactionBadge.tsx

```diff
@@ -0,0 +1,69 @@
+import { Translation, FormattedCryptoAmount } from 'src/components/suite';
+import { WalletAccountTransaction } from 'src/types/wallet';
+import { StakeType } from '@suite-common/wallet-types/src/stake';
+
+import { Button } from '@trezor/components';
+import styled from 'styled-components';
+import { isNetworkSymbol, NetworkSymbol } from '@suite-common/wallet-config';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+import { useSelector } from 'react-redux';
+import { selectSelectedAccount } from 'src/reducers/wallet/selectedAccountReducer';
+import { getInstantStakeType } from 'src/utils/suite/stake';
+import { memo } from 'react';
+
+const Wrapper = styled.div`
+    display: flex;
+    gap: 5px;
+`;
+
+interface TransactionBadgeProps {
+    transaction: WalletAccountTransaction;
+    symbol?: string;
```

Comment:
Can we make it required? Since we check if it's not `undefined` further in the code anyway.